### DISCUSSION TITLE
스터디 2기 두번 째 회고

### DISCUSSION THREAD

**Discussion by SamTheKorean** [DaleStudy/leetcode-study#535] [공지사항]
스터디 2기 참여자 여러분, 안녕하세요.

건강한 스터디 문화를 조성하고 멤버분들과 코치님들께 좋은 경험을 드리기 위해 5주에 한 번씩 회고를 하여 스터디 운영에 반영을 하고 있습니다. 지난 첫 회고에 진행한 것과 같이 6~10주간의 스터디 경험에 대해서 본 디스커션 글에 댓글로 달아주시고, 공감이 되는 다른 댓글에 Upvote를 해주세요. 대댓글로 추가 소통을 해도 좋을 것 같네요. 디스커션에서 Upvote를 많이 받거나 대댓글이 많이 달린 피드백을 위주로 모임 때 함께 얘기해보면 좋을 것 같습니다.

## 댓글 형식
- 좋았던 점과 아쉬웠던 점을 구분하기 위해서 [Went Well] 또는 [To Improve] 말머리를 달아주세요. (예: [Went Well] 단순히 문제만 푸는 것이 아니라 다른 코드를 리뷰하고 답안 설명을 하면서 실력이 늘었습니다.)
- 아쉬웠던 점에 대해서 피드백을 주실 때는 함께 실천해볼 수 잇는 개선 방안도 포함시켜주시면 큰 도움이 될 것 같습니다.
- https://github.com/DaleStudy/leetcode-study/discussions/211 에서 스터디 1기의 회고 기록도 볼 수 있으니 참고 바랍니다.

**Comment by obzva** [comment]
[Went Well]
생존자들이 있으니 힘이 납니다 ㅎㅎ 화이팅
[To Improve]
개인적으로 시간이 부족해져서 1차 회고에서 다짐했던 것만큼 활발하게 리뷰에 참여하지 못하고 있습니다
디스코드 화상 미팅도 참여하지 못하고 있는데, 아쉽습니다
대신에 시간 날 때마다 문제 풀이 자세히 적고, 재밌는 자료들 찾으면 공유하면서 비동기적으로 활동하겠습니다 :)

**Reply by Sunjae95** [reply]
저도 생존자들이 있음에 감사하고 있어요 이제 얼마남지않았어요 화이팅!!

**Comment by Sunjae95** [comment]
[Went Well]
꾸준한 참여로 알고리즘 문제에 대한 두려움이 사라지는거 같아요. 
알고리즘 문제를 처음 공부할 때는, 시작전에 시간이 많이 든다는 생각에 부담감과 스트레스가 있었어요. 
그래서 매번 공부를 중도 포기하는것이 대부분이었어요. 그래서 이번에는 달레스터디의 적절한 문제 난이도 배치, 해설, 스터디원들의 발표에 동기부여가 되고 접근방법이나 풀이법을 알게되어 덕분에 10주차까지 도달 할 수 있었어요. 
남은 5주차에는 무서운 문제들이 많겠지만 10주차까지 풀어온 문제풀이 경험으로 잘 조합하면 해결할 수 있을거 같아요 :)

**Reply by DaleSeo** [reply]
@Sunjae95 님의 피드백은 나중에 웹사이트에 참가자 후기로 넣어도 좋겠네요 😁

**Comment by Sunjae95** [comment]
[To Improve]
얻을게 많은 스터디가 참여도 그래프가 매주 우하향으로 형성되는 것이 너무 아쉬워요. 
부담스럽지 않게 디스코드에 매주 홍보만 하는 정도로만 `리트코드 easy해~`(가제목)라는 운동을 하고싶어요.
스터디를 하는 목표가 중도포기보다는 한문제라도 풀면 어제의 나보다 성장하고 이후 다시 공부할 수 있게 동기부여한다고 생각해요. 
매주 easy 문제가 있으니 easy문제만 풀도록 유도해보는건 어떨까요?

```
예시)
N주차 [문제 링크]easy 문제 너무 깔끔하고 좋은거 같아요.
만약 이 문제를 못풀겠다면 [알고달레링크]에서 참고해서 PR을 올려볼까요~?!
```

**Reply by DaleSeo** [reply]
@Sunjae95 님, 요 아이디어는 제가 잘 이해를 못했는데 모임 때 자세히 설명해주시면 좋을 것 같아요.

**Reply by SamTheKorean** [reply]
디스코드 채널에 격려 메시지가 도움이 되었어서 더 활성화하면 좋겠다는 말씀이었습니다 ㅎㅎ 요것처럼요!
![Screenshot 2024-10-19 at 10 09 02 PM](https://github.com/user-attachments/assets/51fc2cc3-dc22-4b53-940f-fc4b42914421)

**Reply by DaleSeo** [reply]
아하, 네! 디스코드에서 격려 메시지는 멤버들간에 서로 자연스럽게 주고 받으면 될 것 같네요 :)

**Comment by ijlijijij** [comment]
[Went Well]
최근 스터디에 참여하며 코딩 테스트 대부분을 통과하게 되었어요! 예전에는 문제를 풀면 그걸로 끝이었는데, 이제는 매번 다양한 풀이법과 새로운 지식을 배워가고 있어요. 덕분에 한 문제를 풀고 나서도 다른 방법에 대해 고민하는 습관이 생겼습니다.
[To Improve]
최근 바빠져서 급하게 코드만 제출을 하는데, 그래도 끝까지 완주할 수 있도록 힘내겠습니다!

**Reply by DaleSeo** [reply]
@kjb512 님, 와 👏👏👏 코딩 테스트를 대부분 통과하셨다니 뿌듯합니다! 
온라인으로만 꾸준히 활동하시고 정기 모임 때는 한번도 못 뵌 거 같은데 시간되실 때 놀러 오세용~

**Comment by SamTheKorean** [comment]
[Went well]
명절들이 끼어있어서 문제 풀이에 어려움이 있음에도 성실하게 문제풀이 하신 분들 응원합니다!

**Reply by DaleSeo** [reply]
추석, 국군의 날, 개천절, 한글날... 스터디 힘들게 한국엔 휴일이 왜케 많은 거에요? ㅋㅋ

**Comment by SamTheKorean** [comment]
Went well]
문제 별로 설명과 건설적인 토론을 경험하면서 알고리즘에 대한 지식이 확장된다고 느껴서 유익했습니다!

**Comment by TonyKim9401** [comment]
[Went Well] 
9주차 이후가 되면서 Hard 난이도 문제들도 나오고 있는데, Medium 난이도도 힌트가 있어야 겨우겨우 풀어내는지라.. 하지만 꾸준히 하면서 자료구조, 알고리즘, TC, SC에 대해 스스로 찾아보면서 정말 많은 도움이 되고 있습니다. 사람은 많이 줄었지만 그래도 같이 하는 동료가 있다는게 정말 많은 힘이 됩니다. 남은 주차들고 모두 힘내서 진행 했으면 좋겠어요!

**Reply by DaleSeo** [reply]
우리 모범 멤버 @TonyKim9401 님, 문제도 착실히 푸시고 모임 때 발표도 점점 느시고 스터디에서 많은 것을 얻어가고 계신 것 같아서 너무 기쁩니다! 💯

**Comment by HC-kang** [comment]
[Went Well]
다양한 언어로 작성된 다양한 분들의 코드를 보면서 정말 많이 배우고있습니다.
단순히 문제를 푸는 것 뿐 아니라, 사고과정까지 공유해주시는 분들의 글도 많은 도움이 되구요.

그리고 사이드 프로젝트에 참여하면서, 다른 분들은 어떻게 일하시는지도 배울 수 있는 기회가 되는것같습니다.
덤으로 그동안 잘 쓰지 않았던 깃허브의 다양한 기능들도 배우고있습니다!

[To Improve]
1. 저도 다른 분들이 감탄할만한 풀이와 사고과정을 글로 술술 풀어 낼 수 있도록 노력해봐야겠어요. 
정말 아티클 수준으로, 읽기만 해도 뇌리에 쏙쏙 박히는 글을 볼때마다 정말 경탄스럽습니다.

2. 많은 분들이 중도 하차(?) 하셔서 정말 아쉽습니다.
군대에 있을때 항상 전투력 복원의 중요성을 글로만 배웠는데, 저희도 어떻게 하면 마음이 멀어진 동료 분들을 다시 PR로 복귀시킬 수 있을까요..? ㅎㅎ
@Sunjae95 님의 [이런 의견](https://github.com/DaleStudy/leetcode-study/discussions/535#discussioncomment-10954066)처럼, 다소 바빠지신분들을 다시 복돋워 드릴 수 있는 방법이 있으면 좋을 것 같습니다!

**Comment by EgonD3V** [comment]
[Went Well]
알고리즘 문제를 꾸준히 같이 풀 수 있어서 좋았습니다. 매주 할당량이 있으니 계속 풀게되는 점이 좋습니다. 
그리고 좋은 문제들을 큐레이팅 해주셔서 감사합니다.

[To Improve]
토론이 더 활발했으면 좋을 것 같습니다. 문제 외에도 확장된 문제나 새로운 접근법이나, 추가적으로 배울 수 있는 내용이 더 있는데 하지 못한 것 같아 조금 아쉽습니다.
